src/app/external-cluster-wizard/steps/external-cluster/provider/aks/component.ts

```diff
@@ -229,6 +233,13 @@ export class AKSClusterSettingsComponent
     );
   }
 
+  private _getAKSLocations(): void {
+    this._externalClusterService.getAKSLocations().subscribe((locations: AKSLocation[]) => {
+      this.locations = locations.map((location: AKSLocation) => location.name);
+      this.control(Controls.Location).setValue(this.LOCATION_DEFAULT_VALUE);
```

Comment:
Can we add a check here to make sure that `LOCATION_DEFAULT_VALUE` exists in `this.locations`?

Reply:
im not sure what do you mean by `LOCATION_DEFAULT_VALUE` exists in `this.locations`
LOCATION_DEFAULT_VALUE` is a global constant so it's not in  `this.locations`

Reply:
I meant what if the locations fetched from API don't contain `eastus`? Because if it does not, then dropdown options won't contain `eastus` but it will be marked as a selected value.